L1Trigger/ME0Trigger/test/ME0TPEmulator_cfg.py

```diff
@@ -0,0 +1,64 @@
+# Configuration file to unpack CSC digis, run Trigger Primitives emulator,
+# and compare LCTs in the data with LCTs found by the emulator.
+# Slava Valuev; October, 2006.
+
+import FWCore.ParameterSet.Config as cms
+from Configuration.StandardSequences.Eras import eras
+
+process = cms.Process("ME0TPEmulator", eras.Phase2)
+
+process.maxEvents = cms.untracked.PSet(
+  input = cms.untracked.int32(10)
+)
+
+# Hack to add "test" directory to the python path.
```

Comment:
I mean this specific line that is some kind of "hack"

Reply:
I see. It means to get path to this directory generally in any cmssw versions.  Should I remove in a  new pull request ?

Reply:
Why do you need to have the test directory in the python path? This file itself is in the test directory, but it does not appear to import anything else from the test directory.

Reply:
This is from legacy code and legacy code sometime imports stuff  from test directory for convenience.  Since this configuration is a template of using ME0 trigger digi producer,  keeping this line or not should not matter.  But I am fine with a new commit to remove them if necessary.

Reply:
I would prefer to remove it to reduce potential future confusion.

Reply:
just made one commit to fix it.